### PR TITLE
update to observable-process 3.1.0

### PR DESCRIPTION
**Pull request by hugobho**
@kevgo 

#114

### CODE REVIEW

File: features/steps/steps.ls [hunk 1]
```diff
@@ -121,14 +121,16 @@ module.exports = ->
     fs.empty-dir-sync app-dir
     @process = new ObservableProcess(path.join(process.cwd!, 'bin', command),
                                      cwd: app-dir,
-                                     console: dim-console.console)
+                                     stdout: dim-console.stdoutOutput
```
Comment (kevgo) [inline features/steps/steps.ls]: `stdout-output` is a string that contains the cumulated output. You want to use `dimConsole.process.stdout` and `dimConsole.process.stderr` here.

File: features/steps/steps.ls [hunk 2]
```diff
@@ -137,32 +139,37 @@ module.exports = ->
...
+                                     stderr: dim-console.stderrOutput)
 
 
   @When /^executing the abbreviated command ([^"]*) in the terminal$/, (command) ->
     @process = new ObservableProcess(path.join(process.cwd!, 'bin', command),
                                  cwd: app-dir,
-                                 console: off)
+                                 stdout: false
```
Comment (kevgo) [inline features/steps/steps.ls]: Say `off` instead of `false`. Its the same, but semantically more appropriate.

**Comment by hugobho** [comment]
@kevgo

**Comment by kevgo** [comment]
🚢

**Pull request closed without merging**